bears/community/ContributorInitBear.py

```diff
@@ -0,0 +1,24 @@
+from dependency_management.requirements.GemRequirement import GemRequirement
+from dependency_management.requirements.GoRequirement import GoRequirement
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='maintainer',
+        global_bear=True,
+        output_format='regex',
+        output_regex=r'(?P<severity>Error|Warning) \d+ in .+ line '
+                     r'(?P<line>\d+): (?P<message>.*)'
+        )
+class ContributorInitBear:
```

Comment:
@gaocegege Since the `maintainer contributor` command creates an `AUTHORS.md`, this bear should better be named `AuthorsBear`